src/models/user.ts

```diff
@@ -0,0 +1,31 @@
+import db from '../config/database';
+import { Model, DataTypes } from 'sequelize';
+
+export class User extends Model {
```

Comment:
To respect convention, i think it's better to export at the end of the file with a `export = User`. What do you think ?

Reply:
Yes it could be good. We need to change it in other Files so!